### PR TITLE
[docs] Apply marking definition to CSV feeds ingestion (issue/6149)

### PR DESCRIPTION
**Pull request by Goumies**

### CODE REVIEW

File: docs/usage/import-automated.md [hunk 1]
```diff
@@ -96,21 +96,31 @@ CSV feed ingester enables users to import CSV files exposed on URLs. Here's a st
 
 Additional configuration options:
 
-- User responsible for data creation: Define the user responsible for creating data received from this RSS feed. Best practice is to dedicate one user per source for organizational clarity. Please [see the section "Best practices" below](import-automated.md#best-practices-section) for more information.
+- User responsible for data creation: Define the user responsible for creating data received from this CSV feed. Best practice is to dedicate one user per source for organizational clarity. Please [see the section "Best practices" below](import-automated.md#best-practices-section) for more information.
 - Import from date: Specify the date of the oldest data to retrieve. Leave the field empty to import everything.
 
 ![CSV feeds creation: prior CSV mapper test](assets/csv-feeds-creation.png)
```
Comment (Jipegien) [inline docs/usage/import-automated.md:102]: link broken

File: docs/usage/import-automated.md [hunk 2]
```diff
@@ -96,21 +96,31 @@ CSV feed ingester enables users to import CSV files exposed on URLs. Here's a st
...
 
+The CSV mappers allow the user to configure the markings representations. The user can either choose the marking definition levels or use default ones.
```
Comment (Jipegien) [inline docs/usage/import-automated.md:104]:
```suggestion

```

File: docs/usage/import-automated.md [hunk 3]
```diff
@@ -96,21 +96,31 @@ CSV feed ingester enables users to import CSV files exposed on URLs. Here's a st
...
+This setting is took in account and applied globally on every ingested entities.
```
Comment (Jipegien) [inline docs/usage/import-automated.md:105]:
```suggestion
```

File: docs/usage/import-automated.md [hunk 4]
```diff
@@ -96,21 +96,31 @@ CSV feed ingester enables users to import CSV files exposed on URLs. Here's a st
...
+
+Given user choice configuration, the user can select the markings based-on the selected responsible for data creation allowed markings.
```
Comment (Jipegien) [inline docs/usage/import-automated.md:107]:
```suggestion
in CSV Mappers, if you created a representative for Marking definition, you could have chosen between 2 options:

 - let the user choose  marking definitions
 - Use default marking definitions of the user

This configuration applies when using a CSV Mapper for a CSV Ingester. If you select a CSV Mapper containing the option "Use default marking definitions of the user", the default marking definitions of the user you chose to be responsible for the data creation will be applied to all data imported. If you select a CSV Mapper containing the option "let the user choose  marking definitions", you will be presented with the list of all the marking definitions of the user you chose to be responsible for the data creation (and not yours!)
```

**Review by Jipegien — CHANGES_REQUESTED** [review]

File: docs/usage/import-automated.md (hunk 1 above)
Comment (Goumies) [reply docs/usage/import-automated.md]: @Jipegien, in production, the link is working fine : https://docs.opencti.io/latest/usage/import-automated/?h=import#csv-feeds

**Pull request merged**